packages/teams-js/src/internal/utils.ts

```diff
@@ -528,3 +528,17 @@ export function validateUuid(id: string | undefined | null): void {
     throw new Error('id must be a valid UUID');
   }
 }
+
+/**
+ * Cache if performance timers are available to avoid redoing this on each function call.
+ */
+const supportsPerformanceTimers = !!performance && 'now' in performance;
+
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ * @returns current timestamp in milliseconds
+ */
+export function getCurrentTimestamp(): number {
+  return supportsPerformanceTimers ? performance.now() + performance.timeOrigin : new Date().getTime();
```

Comment:
Please add timestamp or date as a return field to support backwards compatibility